src/vs/workbench/contrib/search/browser/media/searchview.css

```diff
@@ -62,15 +62,17 @@
 /* NOTE: height is also used in searchWidget.ts as a constant*/
 .search-view .search-widget .monaco-inputbox > .ibwrapper > textarea.input {
 	overflow: initial;
-	height: 26px; /* set initial height before measure */
+	height: 26px;
+	/* set initial height before measure */
 }
 
 .search-view .search-widget .monaco-findInput .monaco-scrollable-element .scrollbar {
 	opacity: 0;
 }
 
 .search-view .monaco-inputbox > .ibwrapper > textarea.input {
-	scrollbar-width: none; /* Firefox: hide scrollbar */
+	scrollbar-width: none;
+	/* Firefox: hide scrollbar */
```

Comment:
Were these intentional? The comment should be on the same line or above it imo

Reply:
Hmm, the formatter seems to have pushed the comments onto the next line. I'll put them above.